dockstore-webservice/src/main/java/io/dockstore/webservice/resources/ElasticSearchHealthCheck.java

```diff
@@ -0,0 +1,43 @@
+package io.dockstore.webservice.resources;
+
+import javax.ws.rs.core.Response;
+
+import com.codahale.metrics.health.HealthCheck;
+import io.dockstore.webservice.CustomWebApplicationException;
+import io.dockstore.webservice.resources.proposedGA4GH.ToolsExtendedApi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Checks that elastic search is healthy
+ */
+public class ElasticSearchHealthCheck extends HealthCheck {
+    private static final Logger LOG = LoggerFactory.getLogger(ElasticSearchHealthCheck.class);
+
+    private final ToolsExtendedApi toolsExtendedApi;
+
+    public ElasticSearchHealthCheck(ToolsExtendedApi toolsExtendedApi) {
+        this.toolsExtendedApi = toolsExtendedApi;
+    }
+
+    @Override
+    protected Result check() throws Exception {
+        // If elastic search is up with a valid index this should return healthy
+        Response response;
+        try {
+            response = toolsExtendedApi.toolsIndexSearch(null, null, null);
+        } catch (CustomWebApplicationException ex) {
+            LOG.info(ex.getStackTrace().toString());
```

Comment:
You can just throw in the exception rather than get the stack trace directly.
https://www.slf4j.org/api/org/slf4j/Logger.html#info(org.slf4j.Marker,%20java.lang.String,%20java.lang.Throwable)